docs/tutorial/part-one/index.md

```diff
@@ -1,57 +1,57 @@
 ```
 
-When creating a new Gatsby site, you can use the following command structure to create a new site based on any existing Gatsby starter:
+Khi tạo một trang Gatsby mới, bạn cần sử dụng những cấu trục lệnh sau để tạo một trang mới dựa trên bất kì mẫu khởi động Gatsby hiện có:
 
 ```shell
 gatsby new [SITE_DIRECTORY_NAME] [URL_OF_STARTER_GITHUB_REPO]
 ```
 
-If you omit a URL from the end, Gatsby will automatically generate a site for you based on the [**default starter**](https://github.com/gatsbyjs/gatsby-starter-default). For this section of the tutorial, stick with the “Hello World” site you already created in tutorial part zero. You can learn more about [modifying starters](/docs/modifying-a-starter) in the docs.
+Nếu bạn bỏ qua phần URL ở cuối, Gatsby sẽ tự động tạo một trang cho bạn dựa vào [**bản mẫu mặc định**](https://github.com/gatsbyjs/gatsby-starter-default). Đối với phần này của bài hướng dẫn, hãy gắn bó với mẫu "Hello World" mà bạn đã tạo trong bài hướng dẫn phần 0. Bạn có thể học thêm về [modifying starters](/docs/modifying-a-starter) trong tài liệu.
 
-### ✋ Open up the code
+### ✋ Mở code
 
-In your code editor, open up the code generated for your “Hello World” site and take a look at the different directories and files contained in the ‘hello-world’ directory. It should look something like this:
+Trong trình soạn thảo code của bạn, mở phần code được tạo bới trang "Hello World" và xem các thư mục và tệp khác nhau trong thư mục "hello-world". Nó sẽ trông như thế này:
 
-![Hello World project in VS Code](01-hello-world-vscode.png)
+![dự án Hello World trong VS Code](01-hello-world-vscode.png)
 
-_Note: Again, the editor shown here is Visual Studio Code. If you’re using a different editor, it will look a little different._
+_Lưu ý: Một lần nữa, Again, trình soạn thảo hiển thị ở đây là Visual Studio Code. Nếu bạn sử dụng một trình soạn thảo khác, nó sẽ trông hơi khác._
 
-Let’s take a look at the code that powers the homepage.
+Hãy cùng xem phần code tạo nên trang chủ.
 
-> 💡 If you stopped your development server after running `gatsby develop` in the previous section, start it up again now — time to make some changes to the hello-world site!
+> 💡 Nếu như bạn đã dừng máy chủ phát triển sau khi chạy lệnh `gatsby develop` in the previous section, hãy khởi động nó lại bây giờ, đã đến lúc để thực hiện một số thay đổi cho trang hello-world!
 
-## Familiarizing with Gatsby pages
+## Làm quen với các trang Gatsby
 
-Open up the `/src` directory in your code editor. Inside is a single directory: `/pages`.
+Mở thư mục `/src` trong trình soạn thảo code của bạn. Bên trong là một thư mục duy nhất: `/pages`.
 
-Open the file at `src/pages/index.js`. The code in this file creates a component that contains a single div and some text — appropriately, “Hello world!”
+Mở tệp tại `src/pages/index.js`.Phần code trong tệp này tạo ra một thành phần chứa một div và vài văn bản - thích hợp, “Hello world!”
```

Comment:
file -> tập tin
some text -> vài dòng văn bản (trong context này mình thấy dịch như thế này dễ hiểu hơn)
appropraitely -> mình thấy trong bản TV, loại bỏ từ này ra luôn, nó không có ý nghĩa gì nhiều và đọc không được phù hợp lắm
Từ "component" ở đây bạn dùng nguyên bản tiếng ANh giúp mình, chú thích bên cạnh (cụm thành phần). Trong toàn bộ phần còn lại của document này bạn chỉ cần dùng tiếng Anh cho từ này là đủ. Sau này nếu có thay đổi mình sẽ tạo issue sau.